### ISSUE TITLE
Allow pip arguments with dephell jail install

### ISSUE THREAD

**Issue by MrMrRobat** [dephell/dephell#359]
## Short description
Allow use dephell jail to install with extra pip arguments, such as  `--extra-index-url`
```bash
pip install search_stack --extra-index-url=https://pypi.g.smailru.net/devops/openstack-client
```

Setting env variable doesn't help:
```bash
~
$ PIP_EXTRA_INDEX_URL=https://pypi.g.smailru.net/devops/openstack-client dephell jail install search_stack
WARNING cannot find config file
INFO build dependencies graph...
ERROR package not found (package=search-stack, url=https://pypi.org/pypi/search-stack/json)
```

**Comment by orsinium** [comment]
This workaround should work:

```bash
PIP_INDEX_URL="URL" DEPHELL_WAREHOUSE="['URL']" dephell jail install --level=DEBUG PACKAGE
```

However, it's too complicated. Let me rephrase the issue: automatically set `INDEX_URL` for pip based on dephell config.

**Comment by MrMrRobat** [comment]
Even this workaround didn't quite work in my case: just getting exactly same output as in OP

**Comment by jhermann** [comment]
Also, it would be nice if dephell could work with just the "simple" API when installing things – I did not check, but the above seems to imply that you also need the newer warehouse stuff. Services like Artifactory however only offer the "old" simple one.

On a related note, another place to define a local repo is `~/.pip/pip.conf`, in `[global]index-url`.

**Comment by orsinium** [comment]
DepHell supports simple index and switch on it when the server doesn't have http api. The case above is a bug, we're trying to debug it on @MrMrRobat 's environment. With my artifactory (that has only simple index) it works.
